mappings/net/minecraft/advancement/criterion/NetherTravelCriterion.mapping

```diff
@@ -16,4 +16,6 @@ CLASS net/minecraft/class_2108 net/minecraft/advancement/criterion/NetherTravelC
 			ARG 5 exitedY
 			ARG 7 exitedZ
 	FIELD field_9717 ID Lnet/minecraft/class_2960;
-	METHOD method_9080 handle (Lnet/minecraft/class_3222;Lnet/minecraft/class_243;)V
+	METHOD method_9080 trigger (Lnet/minecraft/class_3222;Lnet/minecraft/class_243;)V
+		ARG 1 player
+		ARG 2 entered
```

Comment:
I think `enteredNetherPos` like in the player entity or just `enteredPos` would be more clear